apps/arbeidsforhold-api/src/main/java/no/nav/registre/testnorge/arbeidsforhold/consumer/commnad/CreateArbeidsforholdCommand.java

```diff
@@ -0,0 +1,49 @@
+package no.nav.registre.testnorge.arbeidsforhold.consumer.commnad;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URI;
+import java.util.concurrent.Callable;
+
+import no.nav.registre.testnorge.arbeidsforhold.consumer.dto.ArbeidsforholdDTO;
+import no.nav.registre.testnorge.arbeidsforhold.domain.Arbeidsforhold;
+
+@Slf4j
+@RequiredArgsConstructor
+public class CreateArbeidsforholdCommand implements Callable<Arbeidsforhold> {
+    private final RestTemplate restTemplate;
+    private final String url;
+    private final String token;
+    private final Arbeidsforhold arbeidsforhold;
+
+    @SneakyThrows
+    @Override
+    public Arbeidsforhold call() {
+        RequestEntity<ArbeidsforholdDTO> request = RequestEntity
+                .post(new URI(url + "/v1/arbeidsforhold"))
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + token)
+                .header(AaregHeaders.NAV_CONSUMER_TOKEN, "Bearer " + token)
+                .header(AaregHeaders.NAV_ARBEIDSFORHOLD_KILDEREFERANSE, "testnorge")
+                .body(new ArbeidsforholdDTO(arbeidsforhold));
+
+        log.info("Oppretter arbeidsforhold får {}", arbeidsforhold.getIdent());
```

Comment:
Liten skriveleif "Oppretter arbeidsforhold for {}" og i pakken/mappenavn command